cmd/tailscale/cli/serve.go

```diff
@@ -129,7 +128,7 @@ type localServeClient interface {
 	Status(context.Context) (*ipnstate.Status, error)
 	GetServeConfig(context.Context) (*ipn.ServeConfig, error)
 	SetServeConfig(context.Context, *ipn.ServeConfig) error
-	QueryFeature(context.Context, string) (*tailcfg.QueryFeatureResponse, error)
+	EnableFeatureInteractive(ctx context.Context, feature string, hasRequiredAttrs func(attrs []string) bool) (ok bool, err error)
```

Comment:
if this is the only reason you want to put the code on LocalClient, maybe you can rework the tests a bit instead?

Reply:
Updated